requirements.txt

```diff
@@ -5,7 +5,6 @@
 #
 
 -r ./requirements/requirements-convert.txt
-
```

Comment:
I think it was previously separated because it's included in most of the other requirements files below, to make it stand out.